index.ts

```diff
@@ -69,8 +69,27 @@ async function run() {
     }
 
     case "get": {
+      const [release] = params
       const currentVersion = await readVersion();
-      console.log(currentVersion);
+      if (release) {
```

Comment:
Not a huge deal, but the `if/else` with a `switch` inside could just be flattened to a `switch` with a final `default` case for the else, right?

Reply:
I added the checking similar to the top level switch and so got rid of the switch instead.